File: Tutoriels/Python/POO/Classe_Propriete/src/GPerson.py
#================================================
class GPerson(object): 
    def __init__(self):
        self.__m_name = "Gerard"
        self.__m_email = "gerard@python.com"
    #================================================
    def __getName(self):
        return self.__m_name
    #================================================
    def __setName(self, name):
        self.__m_name = name
    #================================================
    def __getEmail(self):
        return self.__m_email
    #================================================
    def __setEmail(self, email):
        self.__m_email = email
    #================================================
    Name = property(__getName, __setName)
    Email = property(__getEmail, __setEmail)
#================================================

File: Tutoriels/Cpp/POO/Classe/src/GPerson.cpp
//===============================================
#include "GPerson.h"
//===============================================
GPerson::GPerson() {
    m_name = "";
    m_email = "";
}
//===============================================
GPerson::GPerson(const string& name, const string& email) {
    m_name = name;
    m_email = email;
}
//===============================================
GPerson::~GPerson() {

}
//===============================================
void GPerson::print() const {
    string lData = "";
    lData += "Personne: ";
    lData += "Nom: " + m_name + " | ";
    lData += "Email: " + m_email;
    lData += "\n";
    cout << lData.c_str();
}
//===============================================
